packages/module/src/AttachmentEdit/AttachmentEdit.scss

```diff
@@ -0,0 +1,11 @@
+.pf-chatbot__attachment-language {
+  color: var(--pf-t--global--text--color--subtle);
+  font-size: var(--pf-t--global--font--size--100);
```

Comment:
can we find a semantic token that can be used in lieu of this base token with `--100` at the end?

Reply:
My bad! I'm so sorry. I swapped this out and added a comment for the palette token for the color (there's no semantic token with a comparable dark teal that I can find - the closest is 30 and the design called for 50).

Reply:
Sure, let's do it!